libs/dh/charges/feature-prices/src/lib/drawer/charge-content/message-tab/dh-charges-charge-messages-tab.component.ts

```diff
@@ -80,9 +80,10 @@ export class DhChargesChargeMessagesTabComponent implements OnInit, OnChanges, O
 
   @Input() charge: ChargeV1Dto | undefined;
 
-  private transloco = inject(TranslocoService);
-  private chargeMessagesStore = inject(DhChargeMessagesDataAccessApiStore);
-  private dhChargesPricesDrawerService = inject(DhChargesPricesDrawerService);
+  private _transloco = inject(TranslocoService);
+  private _destroyRef = inject(DestroyRef);
+  private _chargeMessagesStore = inject(DhChargeMessagesDataAccessApiStore);
+  private _dhChargesPricesDrawerService = inject(DhChargesPricesDrawerService);
```

Comment:
question: any reason for adding the underscore at the beginning? Is it only to distinguish private properties in the class? Does it matter :)

Reply:
It does not matter, but I think it makes it easier to identify what field is private and what is public. Also they are easier to filter out when using code completion :-)